pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -801,6 +801,7 @@ func getContainer(pid int) (string, error) {
 	}
 
 	// since we use this container for accounting, we need to ensure its a unified hierarchy.
+	// Note: please comment out the following 3 lines if you want create arktos single cluster on Ubuntu 20.04 x86
```

Comment:
It is not ideal to ask dev to comment out the code if they use a different OS. Better detect OS in the code or claim we don't support Ubuntu 20.04 in manual for now.